{{cookiecutter.project_name}}/.github/next_steps/04_zenodo_integration.md

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Next step: Enable Zenodo integration'
+---
+
+By enabling Zenodo integration, your package will automatically get a DOI which can be used to cite your package. After enabling Zenodo integration for your GitHub repository, Zenodo will create a snapshot and archive each release you make on GitHub. Moreover, Zenodo will create a new DOI for each GitHub release of your code.
+
+To enable Zenodo integration:
+
+1. Go to http://zenodo.org and login with your GitHub account. When you are redirected to GitHub, *Authorize application* to give permission to Zenodo to access your account.
+1. Go to <https://zenodo.org/account/settings/github/> and enable Zenodo integration of your repository by clicking on `On` toggle button.
+2. Your package will get a DOI only after you make a release. Create a new release as described in <README.dev.md#33-github>
+3. At this point you should have a DOI. To find out the DOI generated by Zenodo:
+   1. Visit https://zenodo.org/deposit and click on your repository link
+   2. You will find the DOI on the right column (gray-blue badge)
+   3. Click on the badge and copy the **Markdown** badge link
```

Comment:
This is the doi that points to a version, to prevent outdated doi on README use concept doi
```suggestion
   2. You will find the latest DOI in the right column in Versions box in **Cite all versions?** section
   3. Copy the text of the link. For example `10.5281/zenodo.1310751`
```